src/winetricks

```diff
@@ -10212,23 +10220,49 @@ w_metadata galliumnine08 dlls \
 
 load_galliumnine08()
 {
+    w_package_broken "https://github.com/iXit/wine-nine-standalone/issues/149" 8.3
+
     w_download "https://github.com/iXit/wine-nine-standalone/releases/download/v0.8/gallium-nine-standalone-v0.8.tar.gz" 8d73dcf78e4b5edf7a3aea8c339459b5138acd1c957c91c5c06432cb2fc51893
     helper_galliumnine "${file1}"
 }
 
+w_metadata galliumnine09 dlls \
+    title="Gallium Nine Standalone (v0.9)" \
+    publisher="Gallium Nine Team" \
+    year="2023" \
+    media="download" \
+    file1="gallium-nine-standalone-v0.9.tar.gz" \
+    installed_file1="${W_SYSTEM32_DLLS_WIN}/d3d9-nine.dll" \
+    installed_file2="${W_SYSTEM32_DLLS_WIN}/ninewinecfg.exe" \
+    homepage="https://github.com/iXit/wine-nine-standalone"
+
+load_galliumnine09()
+{
+    w_package_broken "https://github.com/iXit/wine-nine-standalone/issues/149" "" 5.7
+
+    w_download "https://github.com/iXit/wine-nine-standalone/releases/download/v0.9/gallium-nine-standalone-v0.9.tar.gz" 0f6826e48cb979bc6d1fb85dbbb9da6025eb364af61f5ee8dbfd0058430778b1
+    helper_galliumnine "${file1}"
+}
+
 w_metadata galliumnine dlls \
     title="Gallium Nine Standalone (latest)" \
     publisher="Gallium Nine Team" \
-    year="2019" \
+    year="2023" \
     media="download" \
     installed_file1="${W_SYSTEM32_DLLS_WIN}/d3d9-nine.dll" \
     installed_file2="${W_SYSTEM32_DLLS_WIN}/ninewinecfg.exe" \
     homepage="https://github.com/iXit/wine-nine-standalone"
 
 load_galliumnine()
 {
-    _W_galliumnine_version="$(w_get_github_latest_release iXit wine-nine-standalone)"
-    w_linkcheck_ignore=1 w_download "https://github.com/iXit/wine-nine-standalone/releases/download/${_W_galliumnine_version}/gallium-nine-standalone-${_W_galliumnine_version}.tar.gz"
+    if w_wine_version_in ,5.7 ; then
+        _W_galliumnine_version="v0.8"
+        w_download "https://github.com/iXit/wine-nine-standalone/releases/download/${_W_galliumnine_version}/gallium-nine-standalone-${_W_galliumnine_version}.tar.gz" 8d73dcf78e4b5edf7a3aea8c339459b5138acd1c957c91c5c06432cb2fc51893
```

Comment:
The 'fix' would be to use 'w_linkcheck_ignore=1' before 'w_download', as the not old case does.

That said, since you're going to hardcode it for older wine (which makes sense), you can skip the variable and put that in the code directly, i.e.,:
`w_download "https://github.com/iXit/wine-nine-standalone/releases/download/v0.8/gallium-nine-standalone-v0.8.tar.gz" 8d73dcf78e4b5edf7a3aea8c339459b5138acd1c957c91c5c06432cb2fc51893`

Reply:
edit: the variable will still be needed, when calling helper_galliumnine(), but you don't need to use the variable for w_download().

Reply:
D'oh, right.
Pushed another solution, which is even more obvious :)